apps/condo/domains/organization/constants/common.js

```diff
@@ -120,6 +121,7 @@ const DEFAULT_ROLES = {
 }
 module.exports = {
     TIN_LENGTH,
+    MIN_NAME_LENGTH,
```

Comment:
Add organization word to thin const name

Reply:
done